nexpose/nexpose_report.py

```diff
@@ -75,24 +70,371 @@ def __init__(self):
         _ReportConfigurationBase.__init__(self)
 
 
-class ReportConfiguration(_ReportConfigurationBase):
+class AdhocReportConfiguration(_ReportConfigurationBase):
+    def __init__(self, template_id, format, site_id=None, owner=None, timezone=None):
+        _ReportConfigurationBase.__init__(self, template_id)
+        self.format = format
+        self.owner = owner
+        self.timezone = timezone
+        self.language = None
+        self.filters = []
+        self.baseline = None
```

Comment:
Why is baseline not passed as a param?

On line 107, I see that there is check to see if it exists, but I can't seem to locate where it gets set.

Reply:
I've been mimicking the ruby gem for these classes and their constructors. I could add more optional arguments I guess, but how many is too many? 🤔